recipes/pycups/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pycups" %}
+{% set version = "2.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycups-{{ version }}.tar.gz
+  sha256: 57434ce5f62548eb12949ca8217f066f4eeb21a5d6ab8b13471dce350e380c90
+
+build:
+  number: 0
+  skip: True  # [not linux]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+    - libcups  # [linux]
```

Comment:
If `libcups` is always needed by the library, probably best to remove these selectors and rely on the `skip` above.

Reply:
Yes, unfortunately pycups does appear to be linux only. I cleaned it up in https://github.com/conda-forge/staged-recipes/pull/15592/commits/d7b7544e4ea42da3d578b9f21eb6a1b297665cff